.github/workflows/test.yaml

```diff
@@ -219,6 +219,9 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
+      with:
+        # spread uses tags as delta reference
+        fetch-depth: 0
```

Comment:
does this slow down the checkout significantly? it probably is not a big deal since any time we lose fetching more git refs will be gained by not uploading more data to the GCE spread workers, but just curious if you have any numbers about this

Reply:
Looking at github timestamps, it's ~20s without the tags vs ~30s with tags. OTOH we need to checkout tags, otherwise there's no ref to generate the deltas for. At the same time, I have no clue how it worked before with delta ref being set to 2.46. The checkout action calls this (notice the --no-tags switch):
```
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +77bb25e402e93f0cd76ce5f1f6e5674730265fcb:refs/remotes/pull/9691/merge
```
While the packed project uploaded by spread was ~470kB, so `git archive` was able to find ref corresponding to 2.46 tag.